scalafix-nsc/src/test/resources/checkSyntax/OrganizeImportsNoGroups.source

```diff
@@ -0,0 +1,28 @@
+rewrites = []
+imports.organize = true
+imports.alwaysUsed = []
```

Comment:
Good point. The defaults are likely to change. I try to keep unit tests as small as possible because large unit test have a tendency to require unnecessary maintenance.